app/models/release_manifest.rb

```diff
@@ -234,7 +234,7 @@ def as_package_json = JSON.parse(content)
   # which could be a digest, version, or tag, and also to request a specific media
   # type in case of multiple manifests per-image.
   def self.find_by_reference!(reference, content_type: nil)
-    base = joins(:release)
+    base = reorder(created_at: :asc).joins(:release)
```

Comment:
Revert?

Reply:
Apparently, Docker is retarded and it doesn't actually do any kind of meaningful content negotiation. Instead, it just asks for anything, `application/vnd.docker.*` and `application/vnd.oci.*`, and in a random order too.

So this needs to be reworked, and how we store and retrieve manifests needs to be more sophisticated, because unless `reference` is a digest, we always want to retrieve the root manifest (i.e. the index for a multi-platform image).